storage/queue.go

```diff
@@ -82,6 +82,24 @@ func (p PeekMessagesParameters) getParameters() url.Values {
 	return out
 }
 
+// UpdateMessageParameters is the set of options can be specified for Update Messsage
+// operation. A zero struct does not use any preferences for the request.
+type UpdateMessageParameters struct {
+	PopReceipt        string
+	VisibilityTimeout int
+}
+
+func (p UpdateMessageParameters) getParameters() url.Values {
+	out := url.Values{}
+	if len(p.PopReceipt) != 0 {
+		out.Set("popreceipt", p.PopReceipt)
+	}
+	if p.VisibilityTimeout != 0 {
```

Comment:
@ahmetalpbalkan, it would be strange that in PutMessageParameters VisibilityTimeout is an int and in UpdateMessageParameters VisibilityTimeout is an *int. However, if you still want it to be that way I can change it. Let me know.